### PR TITLE
added jpa, softDelete, liquibase

### PR DESCRIPTION
**Pull request by TarasYurkovskyi**

### CODE REVIEW

**Review by HovorukhaBohdan — APPROVED** [review]
LGTM

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                  type: varchar(255)
+              - column:
+                  name: cover_image
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```
Comment (d1sam) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml]:
```suggestion
                    nullable: false
                    
```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
```
Comment (d1sam) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
      file: db/changelog/changes/01-create-books-table.yaml
      
```

**Review by d1sam — APPROVED** [review]
good job, but be careful with empty line in the tail of file

**Pull request merged**